.ci.yaml

```diff
@@ -90,6 +90,18 @@ platform_properties:
       device_type: none
       cpu: arm64
       xcode: 14a5294e # xcode 14.0 beta 5
+  mac_benchmark:
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "apple_signing", "version": "version:2022_to_2023"}
+        ]
+      device_type: none
+      mac_model: "Macmini8,1"
+      os: Mac-12
+      tags: >
+        ["devicelab", "hostonly", "mac"]
+      xcode: 14a5294e # xcode 14.0 beta 5
```

Comment:
Should/could we update to xcode 14.2?

Reply:
I would like to update it everywhere but every time we do it causes a huge problem with cache leakage between bots/branches. https://github.com/flutter/flutter/issues/117541
I will tackle it in another PR.